### DISCUSSION TITLE
Error: The pod "Firebase/Firestore" required by the plugin "cloud_firestore" requires a higher minimum iOS deployment version than the plugin's reported minimum version.

### DISCUSSION THREAD

**Discussion by AhmedKhalidOkab** [firebase/flutterfire#12678] [Q&A]
When i run my project in ios  
Show this. 
Error: The pod "Firebase/Firestore" required by the plugin "cloud_firestore" requires a higher minimum iOS deployment version than the plugin's reported
minimum version.
To build, remove the plugin "cloud_firestore", or contact the plugin's developers for assistance.
Error running pod install
Error launching application on iPhone 14 Pro.  
i try a lot to solve this and change version of platform and version of cloud message still same error

**Comment by prabalmaurya08** [comment]
same problem is coming .have you able to solve this??Help me
